src/style.css

```diff
@@ -1 +1,83 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+@media screen {
+  .header {
+    max-width: 1200px;
+    min-width: 1024px;
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+  }
+}
+
+.header__menu {
+  font-family: "Roboto", sans-serif;
```

Comment:
move `font-family` to `html`

Reply:
isn't fixed 
![image](https://user-images.githubusercontent.com/97547824/232205924-17ec4a44-111d-453f-bcea-602c3aebe604.png)